src/servers/main.cc

```diff
@@ -734,6 +759,15 @@ Parse(TRTSERVER_ServerOptions* server_options, int argc, char** argv)
 int
 main(int argc, char** argv)
 {
+#ifdef TRTIS_ENABLE_GPU
+  // Validate the format of specified minimum supported GPU compute capability
+  if (!ValidateMinComputeCapability()) {
+    LOG_ERROR << "TRTIS_MIN_COMPUTE_CAPABILITY should be specified in the "
```

Comment:
What happens if running on CPU-only system? And what if the system contains low C.C. GPU but all models are specified to be on CPU

Reply:
oh, this is just checking the format.